src/helpers/time.ts

```diff
@@ -0,0 +1,21 @@
+import ms from "ms";
+import { getLogger } from "../bindings";
+
+const MILLISECONDS = 1000;
+
+export const convertTimeToSeconds = (time: string): number => {
+  let _time = time;
+
+  // `ms` doesn't support month conversion, so converting to days from months.
+  if (_time.toLowerCase().includes("month")) {
+    _time = "30.4 Days";
```

Comment:
What is the purpose of calculating one month?

Reply:
ms package doesn't have a way to calculate seconds for months.. so if the time is `< 1 Month` I'm calculating the time of an average month.  https://github.com/ubiquity/ubiquibot/pull/478#discussion_r1254956025 ref

Reply:
To answer my own question the purpose of calculating one month is so that users can insert a `Time: 1 Month` label and the bot will understand it. I remember now thanks.